src/styles/theme/oc-button.scss

```diff
@@ -39,17 +69,83 @@
 
   &.uk-button-large {
     min-height: $global-control-large-height;
+    font-size: 1.5rem;
   }
 
   &.uk-button-primary,
   &.uk-button-secondary,
   &.uk-button-danger {
+    border: none;
+
     .oc-icon > svg {
       fill: $global-inverse-color;
     }
   }
+
+  &.uk-button-default {
+    border: 1px solid $action-primary;
+    background-color: transparent;
+    color: $action-primary;
+
+    &:hover,
+    &:focus {
+      border-color: $action-primary-hover;
+      color: $action-primary-hover;
+
+      .oc-icon > svg {
+        fill: $action-primary-hover;
+      }
+    }
+
+    .oc-icon > svg {
+      fill: $action-primary;
+    }
+  }
+
+  &.uk-button-primary {
+    border: none;
+    background-color: $action-primary;
+    box-shadow: 0 4px 6px rgba($shadow-action-primary, .11), 0 1px 3px rgba($shadow-global, .08);
+    color: white;
+
+    &:hover,
+    &:focus {
+      background-color: $action-primary-hover;
+    }
+  }
 }
 
 .oc-button.uk-button-raw {
   @extend .oc-button-reset;
 }
+
+.uk-button-group .oc-button {
+  border-radius: 0;
+  // Prevent negative margin which is coming from UIkit
+  margin-left: 0 !important;
+
+  &:first-of-type {
+    border-radius: 3px 0 0 3px;
+  }
+
+  &:last-of-type {
+    border-radius: 0 3px 3px 0;
+  }
+
+  &.uk-button-default {
+    border-left: none;
+    border-right: none;
+
+    &:first-of-type {
+      border-left: 1px solid $action-primary;
+    }
+  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=5696666)